modules/product/product.service.js

```diff
@@ -51,7 +51,6 @@ class ProductsService {
   }) {
     const isNotBlank = str => !(!str || str.trim().length === 0);
```

Comment:
Do we need to define a function if it is used once?

Reply:
function removed